types/clamscan/package.json

```diff
@@ -7,7 +7,7 @@
     ],
     "dependencies": {
         "@types/node": "*",
-        "axios": "^0.24.0"
+        "axios": "^1.6.0"
```

Comment:
Upstream depends on `"axios": "^0.21.1"`, not `^1.6.0`. I don't think this should be changed.

Reply:
What do you mean by upstream in this context?
I don't think pinning to a version range with a CVE should be desirable

Reply:
`@types/clamscan` describes the npm package `clamscan`. If the types of this package depend on a dependency that is also depended on by `clamscan`, then the two versions should match so they have accurate types. Having `@types/clamscan` depend on axios v1 while `clamscan` itself only uses `0.21` means that the types here would be incorrect.

Reading the package here, I don't even see it depending on `axios`. A better fix would be to drop the dependency here entirely as the package doesn't appear to need it.

Reply:
I see axios v1 in the clamscan dependencies thats why I was asking

https://github.com/kylefarris/clamscan/blob/e04d635a8db1c477047c89a76c24fc581e9ca03f/package.json#L46

But you are totally correct, if axios is not used there shouldn't be a dependency! :)

Reply:
You'd want to check the published package (which is linked in the bot comment): https://unpkg.com/browse/clamscan@2.1.2/package.json

Reply:
Thanks for the clarification.
I see that actually axios is used in the test here. I assume we could move it devDependencies then? With which version?

Reply:
Theoretically we'd keep the same version, as that's what the implementation package depends on. But depending on the test, it may be possible to not depend on axios and write the tests differently.